packages/context/src/binding-inspector.ts

```diff
@@ -272,34 +279,39 @@ function buildBindingKey(
   cls: Constructor<unknown>,
   options: BindingFromClassOptions = {},
 ) {
-  const templateFn = bindingTemplateFor(cls);
-  // Create a temporary binding
-  const bindingTemplate = new Binding('template').apply(templateFn);
-  // Is there a `key` tag?
-  let key: string = options.key || bindingTemplate.tagMap[ContextTags.KEY];
-  if (key) return key;
+  try {
+    const templateFn = bindingTemplateFor(cls);
```

Comment:
I have wrapped most of the code into `try/catch` block, these changes are best viewed with "ignore whitespaces" setting on.

Reply:
Maybe this logic should be moved to `createBindingFromClass()`. 

Is there a better way than mutating the error message?

Reply:
> Maybe this logic should be moved to `createBindingFromClass()`.

That's possible, my solution here is a just a quick hack to give me the information I need for troubleshooting.

> Is there a better way than mutating the error message?

I am not aware of any. It's important to preserve the original stack trace and AFAIK, that requires us to mutate the original error object.

Reply:
See https://github.com/strongloop/loopback-next/pull/3676